extensions/amp-truncate-text/1.0/binary-search.js

```diff
@@ -64,7 +62,10 @@ export function binarySearch(
   stop = BinarySearchStop.IMMEDIATE,
   preference = BinarySearchPreference.NEXT
 ) {
-  devAssert(start <= end);
+  // This currently fails in Preact mode of Storybook. Seems to rely on v0.js
+  // helpers that aren't available to Preact components. Is there an analog
+  // that's available?
+  // devAssert(start <= end);
```

Comment:
`throw` an error instead and add A TODO? WDYT @caroqliu @dvoytenko

Reply:
IMHO no need for assert.

Reply:
Removed the assert for now